config/install/migrate_plus.migration.y_lb_demo_block_cards_item.yml

```diff
@@ -180,8 +180,8 @@ process:
     plugin: default_value
     default_value: card_item
   field_title: title
-  field_body/value: body
-  field_body/format:
+  field_description/value: body
```

Comment:
TO add 
![image](https://github.com/YCloudYUSA/y_lb_demo_content/assets/563412/84f70423-fabb-44c8-8cb2-6aa987d88ef2)